ui/pages/permissions-connect/connect-page/connect-page.tsx

```diff
@@ -291,6 +301,13 @@ export const ConnectPage: React.FC<ConnectPageProps> = ({
     });
   });
 
+  const solanaAccountCreated = useMemo(() => {
```

Comment:
```suggestion
  const existsSolanaAccount = useMemo(() => {
```

Reply:
Updated to `solanaAccountExistsInWallet` here: https://github.com/MetaMask/metamask-extension/pull/31781/commits/b53b22ee44be827a8daa7bc2174448a45133b460